### DISCUSSION TITLE
"Action Required: Fix Renovate Configuration" issues

### DISCUSSION THREAD

**Discussion by artamonovkirill** [renovatebot/renovate#8372] [Request Help]
Starting roughly 2 PM CET, I'm receiving notifications for new issues created by Renovate, like https://github.com/renovatebot/renovate/discussions:
```
Action Required: Fix Renovate Configuration

There is an error with this repository's Renovate configuration that needs to be fixed. As a precaution, Renovate will stop PRs until it is resolved.

Error type: undefined. Note: this is a nested preset so please contact the preset author if you are unable to fix it yourself.
```

Am I the only one getting those?
Is it a problem with my configuration (e.g., a JSON migration I missed a deadline for)?
Or should I raise an issue?

Thanks for any input.

**Comment by viceice** [comment]
see #8371

we are waiting for release: https://github.com/renovatebot/renovate/runs/1741970089?check_suite_focus=true

**Comment by rarkins** [comment]
Sorry, the bad release has been rolled back in the app now

**Comment by artamonovkirill** [comment]
@viceice, @rarkins thanks for the prompt replies and explanation

**Comment by slapierre** [comment]
I have a newbie question: can I just close the issue and move on or will the bot close the issue for me?

**Reply by viceice** [reply]
You can close it. The bot will close it too if you leave it open. 🙃
